### PR TITLE
Reorganized the library

### PR DESCRIPTION
**Pull request by MadAlex1997**
The new structure.

### CODE REVIEW

**Comment by MadAlex1997** [comment]
Oh I also got rid of base because it wasn't doing anything

File: numojo/math/check.mojo [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 import math
 import . _math_funcs as _mf
```
Comment (forFudan) [inline numojo/math/check.mojo:9]: The is a space between `.` and `_math_funcs`. Is this a correct grammar?

**Review by forFudan — APPROVED** [review]

File: numojo/math/check.mojo (hunk 1 above)
Comment (MadAlex1997) [reply numojo/math/check.mojo]: fixed

**Comment by shivasankarka** [comment]
> Oh I also got rid of base because it wasn't doing anything

What about this rotate_bits etc functions? Are those removed?

**Comment by MadAlex1997** [comment]
> > Oh I also got rid of base because it wasn't doing anything
> 
> What about this rotate_bits etc functions? Are those removed?

They are still on main if you want to bring them back.

**Pull request merged**